File: Etud-Avenir/wwwroot/js/types/small-report-dto.ts
export default class SmallReportDTO {
    reportId: number
    quarter: number
    schoolYear: string
    createdAt: Date

    constructor({ reportId, quarter, schoolYear, createdAt }: SmallReportDTO | null) {
        this.reportId = reportId
        this.quarter = quarter
        this.schoolYear = schoolYear
        this.createdAt = createdAt
    }
}

File: Etud-Avenir/DTOs/Report/SmallReportDTO.cs
using System;

namespace Etud_Avenir.DTOs.Report
{
    public class SmallReportDTO
    {
        public int ReportId { get; set; }
        public string SchoolYear { get; set; }
        public int Quarter { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}
